include/gridtools/common/hymap.hpp

```diff
@@ -138,6 +138,25 @@ namespace gridtools {
             };
             using type = keys;
         };
+
+        template <class HyMap>
+        GT_META_DEFINE_ALIAS(to_meta_map,
+            meta::zip,
+            (GT_META_CALL(meta::rename, (meta::list, GT_META_CALL(get_keys, HyMap))),
+                GT_META_CALL(meta::rename, (meta::list, GT_META_CALL(tuple_util::traits::to_types, HyMap)))));
+
+        template <class MetaMap>
```

Comment:
How about that?
```
template
  < class Keys
  , class Values
  , class HymapKeys = GT_META_CALL(meta::rename, (hymap::keys, Keys)) 
  >
GT_META_DEFINE_ALIAS
  ( from_keys_values, meta::rename
  , (HymapKeys::template apply , Values)
);

template
  < class MetaMap
  , class KeysAndValues = GT_META_CALL(meta::transpose, MetaMap)
  , class Keys = GT_META_CALL(meta::first, KeysAndValues)
  , class Values = GT_META_CALL(meta::second, KeysAndValues)
  >
GT_META_DEFINE_ALIAS(from_meta_map, from_keys_values, (Keys, Values));
```

Reply:
Looks good, from_keys_values is probably useful, will add this tomorrow.